.gitignore

```diff
@@ -2,3 +2,4 @@ hide-*.js
 node_modules
 .idea/
 .DS_Store
+package-lock.json
```

Comment:
Please don't include this change in this PR. It is irrelevant to the addition of the support for Io.